node/src/chain_spec.rs

```diff
@@ -87,6 +87,7 @@ pub fn development_config() -> Result<ChainSpec, String> {
         None,
         // Protocol ID
         None,
+        None,
```

Comment:
I think it would be useful for future to left the comment what this None actually is, as it done around

Reply:
Same for below